### PR TITLE
[DA-84] implement CLI options for reports

### PR DESCRIPTION
**Pull request by janinko**

### CODE REVIEW

**Comment by thescouser89** [comment]
retest this please

File: cli/bwlist-admin.sh [hunk 1]
```diff
@@ -16,6 +16,10 @@ printUsage() {
     echo "    List all artifacts in black or white list"
     echo "$0 pom [--no-transitive]";
     echo "    Check all dependencies from pom in working directory (using dependency:list) and print their Black/White list status"
+    echo "$0 lookup";
+    echo "    Read G:A:Vs from standart input and finds coresponding redhat versions"
```
Comment (thescouser89) [inline cli/bwlist-admin.sh]: s/standart/standard
s/coresponding/corresponding

File: cli/bwlist.sh [hunk 2]
```diff
@@ -12,6 +12,10 @@ printUsage() {
     echo "    List all artifacts in black or white list"
     echo "$0 pom [--no-transitive]";
     echo "    Check all dependencies from pom in working directory (using dependency:list) and print their Black/White list status"
+    echo "$0 lookup";
+    echo "    Read G:A:Vs from standart input and finds coresponding redhat versions"
```
Comment (thescouser89) [inline cli/bwlist.sh]: s/standart/standard
s/coresponding/corresponding

**Comment by thescouser89** [comment]
I find it a little bit weird that the `lookup` option is the only one that expects values from standard input, whereas for the others we can specify the GAV as a cli parameter.

File: cli/listings.sh [hunk 3]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-target=10.3.10.85:8080/da/rest/listings
+target=10.3.10.85:8080/da/rest
```
Comment (thescouser89) [inline cli/listings.sh]: Hardcoded ip address. Not sure what would be a better alternative though.

**Comment by thescouser89** [comment]
I've tried:
```bash
➜  cli git:(d1d1b5e) ✗ echo "xom:xom:1.2.7" | ./bwlist.sh lookup
[DEBUG-dcheung]  [ {"groupId":"xom", "artifactId":"xom", "version":"1.2.7"} ]
Traceback (most recent call last):
  File "./pretty-lookup.py", line 15, in <module>
    if e["blacklisted"] and e["whitelisted"]:
KeyError: 'blacklisted'
➜  cli git:(d1d1b5e) ✗ 
```

Note: I modified `listings.sh` to use my local PNC instance where the lookup endpoint was confirmed to work via swagger-ui.

File: cli/listings.sh (hunk 3 above)
Comment (janinko) [inline cli/listings.sh]: I also don't know about better alternative.

Comment (jbartece) [inline cli/listings.sh]: I think it is not important for now and it will be easier for users to use the tool (they do not need to know the url)

**Comment by janinko** [comment]
> I find it a little bit weird that the lookup option is the only one that expects values from standard input, whereas for the others we can specify the GAV as a cli parameter.

The lookup option is only option that can take multiple GAVs and I'm not fond of having 
```./bwcli.sh lookup G:A:V G:A:V G:A:V G:A:V ...```
Maybe if there would be one optional GAV and if not provided then it would read from stdin?
```./bwlist.sh lookup [GROUP_ID:ARTIFACT_ID:VERSION]
    Finds corresponding redhat versions for Read G:A:V. If the optional parameter is not specified, it reads G:A:Vs from standard input and finds corresponding redhat versions for all of them.

**Comment by jbartece** [comment]
:+1:

**Comment by thescouser89** [comment]
Woops I knew I should have posted before. The lookup command is still throwing some errors, will fix it in another PR!

**Pull request merged**